.github/workflows/ci.yml

```
@@ -1,5 +1,10 @@
 name: CI
-on: [push]
+
+on:
+  pull_request:
+    branches:
+      - main
+  push:
```

Maybe let's do

```suggestion
on:
  # We would like to trigger for CI for any pull request action -
  # both from QuantCo's branches as well as forks.
  pull_request:
  # In addition to pull requests, we want to run CI for pushes
  # to the main branch and tags.
  push:
    branches:
      - "main"
    tags:
      - "*"
```

instead to be consistent with [glum](https://github.com/Quantco/glum/blob/b471522c611b263c00ae841fd0f46660c31a6d5f/.github/workflows/ci.yml#L2-L12) and [metalearners](https://github.com/Quantco/metalearners/blob/250a0c01a1d1442488a07c90bb55540931336ad8/.github/workflows/ci.yml#L2-L12). I believe this avoids running the CI twice when a PR is submitted from a tabmat branch to main.